src/dao/api.dao.ts

```diff
@@ -28,7 +28,7 @@ export class ClaimDao {
     const { name, images, ...rest } = rawClaim;
     const createdClaim = await prisma.claim.create({
       data: {
-        issuerId: `http://trustclaims.whatscookin.us/users/${userId}`,
+        issuerId: `${process.env.BASE_URL}/users/${userId}`,
```

Comment:
Nice.
With this development, once PR is merged you need to tell the team of the change in the env and also shared the name and value.